tinygrad/ops_cpu.py

```diff
@@ -35,8 +35,7 @@ def backward(ctx, grad_output):
 class Mul(Function):
   @staticmethod
   def forward(ctx, x, y):
-    ctx.save_for_backward(x, y)
-    return x*y
+    return (lambda x,y: x*y)(*ctx.save_for_backward(x, y))
```

Comment:
making the `save_for_backwards` return it's params made it possible to do some lambda and decomposition trickery to reduce line counts

Reply:
this hurts readablity

Reply:
It kinda just seems like you don't like lambdas, these seem super readable to me